packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -327,18 +332,27 @@ class AndroidCameraCameraX extends CameraPlatform {
     CameraDescription cameraDescription,
     MediaSettings? mediaSettings,
   ) async {
+    // TODO(bparrishMines): Failure to get permissions isn't actually handled.
```

Comment:
Meaning this will just crash?

Reply:
I looked at the original code again and realized that a `CameraException` was thrown at the API level. I updated it to throw in this method since this is no longer possible in this implementation.